mesheryctl/internal/cli/root/system/start.go

```diff
@@ -322,22 +322,41 @@ func start() error {
 			return err
 		}
 
-		// fetch the manifest files corresponding to the version specified
-		manifests, err := utils.FetchManifests(currCtx)
-
-		if err != nil {
-			return err
-		}
-		// path to the manifest files ~/.meshery/manifests
-		manifestFiles := filepath.Join(utils.MesheryFolder, utils.ManifestsFolder)
-
-		// change version in meshery-deployment manifest
-		err = utils.ChangeManifestVersion(utils.MesheryDeployment, channel, version, filepath.Join(manifestFiles, utils.MesheryDeployment))
-		if err != nil {
-			return err
+		var manifests []utils.Manifest
+		// check if skipUpdate flag is used
+		// if it is, check if cached manifests can be used
+		if skipUpdateFlag {
+			err = utils.CanUseCachedManifests(currCtx)
+			if err != nil {
+				return errors.Wrap(err, "cannot start Meshery")
+			}
+			for _, adapter := range currCtx.GetAdapters() {
+				var temp utils.Manifest
+				temp.Path = adapter + "-deployment.yaml"
+				manifests = append(manifests, temp)
+			}
+		} else {
+			// fetch the manifest files corresponding to the version specified
+			manifests, err = utils.FetchManifests(currCtx)
```

Comment:
@metonymic-smokey, this would fetch all manifests except operator manifests, right?

Reply:
@hexxdump yes
The operator manifests are fetched later.

Reply:
I have changed the new function and removed the call to check for cached operator manifests, since it's being checked later.